src/lib/components/account/sendVerificationEmailModal.svelte

```diff
@@ -1,32 +1,110 @@
 <script lang="ts">
-    import { invalidate } from '$app/navigation';
+    import { invalidate, goto } from '$app/navigation';
     import { Modal } from '$lib/components';
     import { Button } from '$lib/elements/forms';
-    import { addNotification } from '$lib/stores/notifications';
     import { sdk } from '$lib/stores/sdk';
     import { user } from '$lib/stores/user';
     import { get } from 'svelte/store';
     import { page } from '$app/state';
+    import Link from '$lib/elements/link.svelte';
     import { Card, Layout, Typography } from '@appwrite.io/pink-svelte';
     import { Dependencies } from '$lib/constants';
-    import { onMount } from 'svelte';
+    import { onMount, onDestroy } from 'svelte';
+    import { base } from '$app/paths';
+    import { browser } from '$app/environment';
+    import { addNotification } from '$lib/stores/notifications';
 
-    let { show = $bindable(false) } = $props();
+    let { show = $bindable(false), email }: { show?: boolean; email?: string } = $props();
     let creating = $state(false);
     let emailSent = $state(false);
+    let resendTimer = $state(0);
+    let timerInterval: ReturnType<typeof setInterval> | null = null;
+
+    async function logout(redirect = true) {
+        try {
+            await sdk.forConsole.account.deleteSession({ sessionId: 'current' });
+            if (redirect) {
+                await invalidate(Dependencies.ACCOUNT);
+                goto(`${base}/login`);
+            }
+        } catch (error) {
+            addNotification({
+                type: 'error',
+                title: 'Logout failed',
+                message: 'Unable to log out. Please try again or refresh the page.'
+            });
+        }
+    }
 
-    let cleanUrl = $derived(page.url.origin + page.url.pathname);
+    const cleanUrl = $derived(page.url.origin + page.url.pathname);
```

Comment:
It seems odd to end up on this page that tells you to verify your email again. Can we have the page be something that says you've been verified or if there was an error and you can close the page?

Reply:
we now check the verification on the `+page.ts` so if verified, you get redirected to base. Previously this page still showed `verify email` modal, should be fixed now.